src/test/java/uk/gov/hmcts/reform/sendletter/SampleData.java

```diff
@@ -27,6 +28,20 @@ public static Letter letter() throws IOException {
         );
     }
 
+    public static uk.gov.hmcts.reform.sendletter.entity.Letter letterEntity(String service) {
+        try {
+            return new uk.gov.hmcts.reform.sendletter.entity.Letter(
+                "messageId",
+                service,
+                new ObjectMapper().readTree("{}"),
+                "a type",
+                new byte[1]
+            );
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
is this just to make errors unchecked?

Reply:
Yes, I didn't want to have a checked IOException on this method since there's nothing tests could do about it than fail. Do you think it makes sense to do this?

Reply:
less confusing in tests, more space to name the test case. all good